### PR TITLE
Components/filter bar

### PR DESCRIPTION
**Pull request by ayusrina**

### CODE REVIEW

File: src/components/FilterBar.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, {useState} from 'react';
+import { DatePicker, Space } from 'antd';
+import moment from 'moment';
+
+
+export default function FilterBar(onChange) {
```
Comment (yulyulyharuka) [inline src/components/FilterBar.js]: export default function FilterBar(props)

File: src/components/FilterBar.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    console.log(JSON.stringify(newData), "filter data")
+    setDataFilter(newData)
+  }
+
+  const [list] = useState([]);
+
+  return (
+      <div>
+        <Space direction="vertical" size={12}>
+          <DatePicker onChange={onChange} defaultValue={moment()} />
```
Comment (yulyulyharuka) [inline src/components/FilterBar.js]: <DatePicker onChange={props.onChange} defaultValue={moment()} />

Comment (yulyulyharuka) [reply src/components/FilterBar.js]: it supposes to be <DatePicker onChange={props.onChange} defaultValue={moment()} />

Comment (yulyulyharuka) [reply src/components/FilterBar.js]: onChange={onChange}

**Review by yulyulyharuka — CHANGES_REQUESTED** [review]

File: src/components/FilterBar.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+export default function FilterBar(props) {
+  const [startDate, setStartDate] = useState(moment())
+  const [dataFilter, setDataFilter] = useState([])
+
+  function onChange(date) {
+    setStartDate(date)
+    const newData = list.filter(item => moment(item.created_at.substring(0, 10)).isSame(date, 'date'))
+    console.log(JSON.stringify(newData), "filter data")
```
Comment (sayaandreas) [inline src/components/FilterBar.js]: hapus aja jah console nya

**Review by sayaandreas — CHANGES_REQUESTED** [review]

**Review by sayaandreas — APPROVED** [review]

**Review by yulyulyharuka — APPROVED** [review]

**Pull request merged**